crates/anvil/src/eth/backend/mem/mod.rs

```diff
@@ -676,6 +676,10 @@ impl Backend {
         Ok(self.db.write().await.revert(id))
     }
 
+    pub async fn list_snapshots(&self) -> HashMap<U256, (u64, H256)> {
```

Comment:
This function doesn't have to be async

Reply:
indeed. the await was refactored away at some point. fixed